torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -947,7 +947,7 @@ TensorView* sum_to(TensorView* in, const std::vector<Int*>& sum_to_size) {
   bool reduction_within_shape = false;
 
   // Reduce rest of the dims with keep_dim
-  for (int i = leading_dims; i < root.size(); i++) {
+  for (int i = leading_dims; i < (int)root.size(); i++) {
```

Comment:
please use the "functional" syntax (`int(root.size())`) instead, or `static_cast<int>(root.size())` - I'd prefer the former